pkgs/tools/misc/lokalise2-cli/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "lokalise2-cli";
+  version = "2.5.1";
+
+  goPackagePath = "github.com/lokalise/lokalise-cli-2-go";
+
+  src = fetchFromGitHub {
+    owner = "lokalise";
+    repo = "lokalise-cli-2-go";
+    rev = "v${version}";
+    sha256 = "099cyf9ipskfr4alcrbpvmsci5mz8rhb9ibbmhhqwb1fks330d8l";
+  };
+
+  vendorSha256 = "09qkh9liba4g8qakzgsszzf9lsg1k8x6biny868d390crp8dn1ac";
+
+  postInstall = ''
+    mv $out/bin/lokalise-cli-2-go $out/bin/lokalise2
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://lokalise.com";
+    description = "Translation platform for developers. Upload language files, translate, integrate via API.";
```

Comment:
The folks at Lokalise have yet to provide a license lokalise/lokalise-cli-2-go#38 :shrug: